app/templates/components/event-card.hbs

```diff
@@ -23,12 +23,12 @@
       </SmartOverflow>
       <div class="meta">
         <span class="date">
-          {{#if (eq (general-date this.event.startsAt 'date-short') (general-date this.event.endsAt 'date-short')) }}
+          {{#if this.isSingleDay}}
             {{general-date this.event.startsAt 'date-time-long'}}
```

Comment:
pls understand passing tz is not to show tz, it is used in general-date to get time in particular tz

Reply:
Thanks for clarifying.